R/system.R

```diff
@@ -658,7 +658,16 @@ getAmazonAthenaConnection <- function(driver = "", region = "", authenticationTy
   if (is.null(conn)) {
     #https://www.rdocumentation.org/packages/RODBC/versions/1.3-15/topics/odbcConnect
     #For better performance, set max value (i.e. 1024) for rows_at_time
-    conn <- RODBC::odbcDriverConnect(connection = connectionString, rows_at_time = 1024)
+    conn <- DBI::dbConnect(
```

Comment:
Add comment on why we are still generating connection string above even though we don't pass it to this call.

Reply:
Added comment.